repolinter-rulesets/nr1-lib-deprecations.yml

```diff
@@ -0,0 +1,116 @@
+# Schema: https://raw.githubusercontent.com/newrelic-forks/repolinter/master/rulesets/schema.json
+version: 2
+axioms: {}
+rules:
+  nr1-lib-deprecation-dropdown-label:
+    level: error
+    rule:
+      type: file-not-contents
+      nocase: true
+      options:
+        globsAll:
+        - "**/*.@(js|ts|jsx|tsx)"
+        fail-on-non-exist: false
+        flags: gm
+        content: (<(Dropdown)\s[^>]*(label)=*)
+        human-readable-content: deprecated Dropdown label property
+    policyInfo: >-
+      See the New Relic Developer docs for current API details
+      https://developer.newrelic.com/components/dropdown
+    policyUrl: https://github.com/newrelic/developer-website/issues/1033
+  nr1-lib-deprecation-accountpicker-onchange:
+    level: warning
```

Comment:
Should this be `warning` or `error`?

Reply:
I'd vote for an `error` since it's no longer supported as of Mar 1.

Reply:
Reason I went with warning is that (as @jbeveland27 highlighted), this rule is not identifying for certain that the second parameter is being used. Strictly speaking, that's something I could condense into a regex. The handler could all a function that had been passed down. So, I'm highlighting anywhere that I see the Component and attribute to ask the developer to validate that their code is not invalid. Make sense @jpvajda?